nautobot/docs/additional-features/jobs.md

```diff
@@ -568,9 +595,12 @@ Once a job has been run, the latest [`JobResult`](../models/extras/jobresult.md)
 
```

Comment:
Need to update the docs on `class_path` just above this point, I missed this in #3840.